packages/server/graphql/types/AtlassianIntegration.ts

```diff
@@ -182,20 +184,18 @@ const AtlassianIntegration = new GraphQLObjectType<any, GQLContext>({
       type: GraphQLNonNull(GraphQLList(GraphQLNonNull(JiraSearchQuery))),
       description:
         'the list of suggested search queries, sorted by most recent. Guaranteed to be < 60 days old',
-      resolve: async ({id: atlassianAuthId, jiraSearchQueries}: AtlassianAuth) => {
+      resolve: async ({teamId, userId, jiraSearchQueries}) => {
         const expirationThresh = ms('60d')
         const thresh = new Date(Date.now() - expirationThresh)
-        const searchQueries = jiraSearchQueries || []
-        const unexpiredQueries = searchQueries.filter((query) => query.lastUsedAt > thresh)
-        if (unexpiredQueries.length < searchQueries.length) {
-          const r = await getRethink()
-          await r
-            .table('AtlassianAuth')
-            .get(atlassianAuthId)
-            .update({
-              jiraSearchQueries: unexpiredQueries
-            })
-            .run()
+        const unexpiredQueries = jiraSearchQueries.filter(
+          (query) => new Date(query.lastUsedAt) > thresh
```

Comment:
beware, previously with data that came from rethink `lastUsedAt` was `Date`, with PG now it's a string so the old comparison didn't work.

Reply:
I'm pretty sure it's because underlying PG type is simply `JSONB` not the `Date`